Integrations/integration-SymantecEndpointProtection.yml

```diff
@@ -52,6 +89,16 @@ script:
         }
     }
 
+    var buildClientXml = function(data) {
+        return '<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:cli="http://client.webservice.sepm.symantec.com/"> \
+                <soapenv:Header/><soapenv:Body>{0}</soapenv:Body></soapenv:Envelope>'.format(data);
+    }
+
+    var buildCommandXml = function(data) {
+        return '<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:com="http://command.client.webservice.sepm.symantec.com/"> \
+                <soapenv:Header/><soapenv:Body>{0}</soapenv:Body></soapenv:Envelope>'.format(data);
+    }
+
     var parseResponse = function(resp) {
       if (resp.StatusCode === 200) {
           try {
```

Comment:
Some of the commands of the API send JSON (e.g. "authenticate" or "computers") and some send XML (e.g. the Command ones). However, I'm parsing the result as JSON anyway and handle it in a united way.